system-addon/content-src/components/Card/_Card.scss

```diff
@@ -60,6 +65,7 @@
   }
 
   .card-preview-image-outer {
+    background-color: $grey-30;
```

Comment:
Some reason I thought we had some gradient placeholder.. but I can't seem to find it even quickly looking through old versions.